websites/I/IBL/presence.ts

```diff
@@ -0,0 +1,149 @@
+const presence = new Presence({
+    clientId: '815553000470478850'
+}), timestamp = Math.floor(Date.now() / 1000);
+
+presence.on('UpdateData', async () => {
+    const showTimestamp: boolean = await presence.getSetting('timestamp'),
+    showButtons: boolean = await presence.getSetting('buttons'),
+    presenceData: PresenceData = {
+        largeImageKey: 'logo',
+        startTimestamp: timestamp
+    },
+
+        loc = document.location.pathname;

```

Comment:
Use some appropriate variable names please
```suggestion
    },
        { pathname } = document.location;
```